internal/command/apps/create.go

```diff
@@ -47,14 +47,11 @@ may be fetched with 'fly config save -a <app_name>'`
 			Name:        "network",
 			Description: "Specify custom network id",
 		},
-		flag.Bool{
-			Name:        "machines",
-			Description: "Use the machines platform",
-		},
```

Comment:
Can we make it a no-op? I am afraid it could break some pipelines

Reply:
What do you mean?

Reply:
Yeah, good call!